profile/GbaSpecimen.StructureDefinition.json

```diff
@@ -0,0 +1,268 @@
+{
+  "resourceType": "StructureDefinition",
+  "url": "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaSpecimen",
+  "name": "GbaSpecimenTissue",
+  "status": "draft",
+  "fhirVersion": "4.0.0",
+  "kind": "resource",
+  "abstract": false,
+  "type": "Specimen",
+  "baseDefinition": "http://hl7.org/fhir/StructureDefinition/Specimen",
+  "derivation": "constraint",
+  "differential": {
+    "element": [
+      {
+        "id": "Specimen.extension",
+        "path": "Specimen.extension",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "url"
+            }
+          ],
+          "rules": "open"
+        }
+      },
+      {
+        "id": "Specimen.extension:longTermStorageTemperature",
+        "path": "Specimen.extension",
+        "sliceName": "longTermStorageTemperature",
+        "type": [
+          {
+            "code": "Extension",
+            "profile": [
+              "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaExtensionStorageTemperature"
+            ]
+          }
+        ]
+      },
+      {
+        "id": "Specimen.extension:diagnosis",
+        "path": "Specimen.extension",
+        "sliceName": "diagnosis",
+        "type": [
+          {
+            "code": "Extension",
+            "profile": [
+              "http://fhir.germanbiobanknode.de/fhir/StructureDefinition/GbaExtensionDiagnosis"
+            ]
+          }
+        ]
+      },
+      {
+        "id": "Specimen.identifier",
+        "path": "Specimen.identifier",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.accessionIdentifier",
+        "path": "Specimen.accessionIdentifier",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.status",
+        "path": "Specimen.status",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.type",
+        "path": "Specimen.type",
+        "binding": {
+          "strength": "required",
+          "valueSet": "http://fhir.germanbiobanknode.de/fhir/ValueSet/GbaValueSetSampleType"
+        }
+      },
+      {
+        "id": "Specimen.type.coding.version",
+        "path": "Specimen.type.coding.version",
+        "max": "0"
+      },
+      {
+        "id": "Specimen.type.coding.code",
+        "path": "Specimen.type.coding.code",
+        "code": [
+          {
+            "code": "Tissue",
+            "display": "Tissue"
+          },
+          {
+            "code": "Tissue formalin",
+            "display": "Tissue formalin"
+          },
+          {
+            "code": "Tissue frozen",
+            "display": "Tissue frozen"
+          },
+          {
+            "code": "Tissue Paxgene or else",
+            "display": "Tissue Paxgene or else"
+          },
+          {
+            "code": "Other tissue storage",
+            "display": "Other tissue storage"
+          }
+        ]
+      },
```

Comment:
That looks wrong. We have already the ValueSet binding.

Reply:
Right! Removed it.